docs/self-managed/platform-deployment/helm-kubernetes/overview.md

```diff
@@ -11,16 +11,20 @@ There are many ways you can provision and configure a Kubernetes cluster, and th
 
 ## Kubernetes environments
 
-You can install Camunda 8 on your Kubernetes environment of choice, e.g.:
+You can install Camunda 8 on your [CNCF Certified Kubernetes](https://www.cncf.io/training/certification/software-conformance/) environment of choice, e.g.:
 
-- [Stock Kubernetes](https://kubernetes.io/docs/).
 - [Kubernetes KIND](https://github.com/kubernetes-sigs/kind), Minikube, K3s, and MicroK8s for local development.
-- [Red Hat OpenShift](https://www.redhat.com/en/technologies/cloud-computing/openshift), an enterprise ready solution with a focus on security.
-- Cloud service providers like Google GKE, Azure AKS, Amazon EKS, etc.
+- [Red Hat OpenShift](./platforms/redhat-openshift.md), an enterprise ready solution with a focus on security.
+- Cloud service providers:
+  - [Amazon EKS](./platforms/amazon-eks/amazon-eks.md)
+  - [Google GKE](./platforms/google-gke.md)
+  - [Azure AKS](./platforms/microsoft-aks.md)
 
 ## Officially supported platforms
 
-With the right configuration, Camunda 8 Self-Managed can be deployed on any Kubernetes distribution (Cloud or on-premise). However, we officially test and support a [specific list of platforms](./platforms/platforms.md).
+With the right configuration, Camunda 8 Self-Managed can be deployed on any Kubernetes distribution (Cloud or on-premise). 
+- Camunda 8 self-managed supports the following [CNCF Certified Kubernetes distributions](https://www.cncf.io/training/certification/software-conformance/). Some distributions include custom patches or upstream source code changes. Camunda will not provide support for issues arising from these changes.
```

Comment:
I think above statement is bit confusing and contradicting. First it says that it supports the CNCF certified K8s Distro but then it also says Camunda will not provide support for issues arising from these changes.

1. Can we move this part to another section may be something like "Compatible Platforms” and keep only tested platforms under "Officially supported platforms” section for better clarity?
2. Does it make sense to change the statement to following? 
 `Camunda 8 self-managed is compatible with the following [CNCF Certified Kubernetes distributions](https://www.cncf.io/training/certification/software-conformance/).”`

Reply:
This is a pretty standard statement. While certified distribution may be fully working, the service or a vendor may add extra items that cause new bugs/issues, and this statement protects us from these eventualities.